### PR TITLE
Add conuse

### PR DESCRIPTION
**Pull request by monkindey**
Inspired by constate and unstated-next

### CODE REVIEW

**Comment by monkindey** [comment]
> Can you sort it alphabetically?

OK

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@
 ## Packages
 
 - [`@21kb/react-hooks`](https://github.com/21kb/react-hooks) A set of React Hooks to get _hooked_ on.
+- [`conuse`](https://github.com/fodau/conuse) Share Hook with Context
```
Comment (aarongarciah) [inline README.md]: Should be placed after `constate`.

Comment (monkindey) [reply README.md]: Addressed it

**Comment by aarongarciah** [comment]
Thanks!

**Pull request merged**